src/styles/Main/card.scss

```diff
@@ -0,0 +1,94 @@
+@import "../utils/variables.scss";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 166px;
+  padding: 16px;
+  padding-top: 32px;
+  font-family: "Roboto", Helvetica, sans-serif;
+  background-color: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    object-fit: contain;
+    object-position: center;
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $PrimaryColor;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $SecondaryColor;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 0;
+    margin-bottom: 24px;
+  }
+
+  &__reviewsNumber {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $PrimaryColor;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__priceTitle {
+    font-weight: 400;
```

Comment:
400 s a default `font-weight`

Reply:
and let's use bem naming convention, since we use bem
<img width="728" alt="image" src="https://user-images.githubusercontent.com/102411130/222069033-7f703218-25c6-4d38-830f-858318058b49.png">